### ISSUE TITLE
Deprecation warnings

### ISSUE THREAD

**Issue by bendem** [bendews/ansible-letsencrypt-cloudflare#1]
```
TASK [letsencrypt_cloudflare : initiate the Let's Encrypt challenge] **************************************************************************************************************************
task path: /Users/benjamin/.ansible/roles/letsencrypt_cloudflare/tasks/main.yml:83
[DEPRECATION WARNING]: The 'letsencrypt' module is being renamed 'acme_certificate'. This feature will be removed in version 2.10. Deprecation warnings can be disabled by setting
deprecation_warnings=False in ansible.cfg.
```

```
TASK [letsencrypt_cloudflare : validate the Let's Encrypt challenge] **************************************************************************************************************************
task path: /Users/benjamin/.ansible/roles/letsencrypt_cloudflare/tasks/main.yml:111
[DEPRECATION WARNING]: Using tests as filters is deprecated. Instead of using `result|success` use `result is success`. This feature will be removed in version 2.9. Deprecation warnings can
be disabled by setting deprecation_warnings=False in ansible.cfg.
```

**Comment by bendews** [comment]
Fixed on latest commit (Release v1.1). Massive thanks for the feedback! 👍

**Comment by bendem** [comment]
Awesome, thanks for the quick fix!
